app/services/contracted-hours-service.js

```diff
@@ -35,10 +36,14 @@ module.exports.updateContractedHours = function (id, organisationLevel, hours) {
     if (count === 0) {
       throw new Error('Offender manager with id: ' + id + ' has not had contracted hours updated')
     }
-    return getLatestIdsForWorkloadPointsRecalc(id)
-    .then(function (ids) {
-      return createWorkloadPointsRecalculationTask(ids.workloadStagingId, ids.workloadReportId, 1)
-    })
+    if (workloadType === workloadTypes.PROBATION) {
+      return getLatestIdsForWorkloadPointsRecalc(id)
+      .then(function (ids) {
+        return createWorkloadPointsRecalculationTask(ids.workloadStagingId, ids.workloadReportId, 1)
+      })
+    } else {
+      // TODO: kick off recalc of court-report-workload-points
```

Comment:
We don't need to wait until the worker is implemented in order to create this task.

Reply:
Added the task and associated data services